.github/workflows/update-site.yaml

```diff
@@ -0,0 +1,23 @@
+name: Update Site
```

Comment:
This doesn't need to be added.

Reply:
file can be removed/removed in the reference fork.